src/help/components/contextual/SideButtons.tsx

```diff
@@ -17,40 +17,36 @@ const SideButtons = ({ displayHelp, onClick }: Props) => {
   const [displayFeedback, setDisplayFeedback] = useState(false);
 
   useEffect(() => {
-    // Checking if there is a scroll bar
+    // // Checking if there is a scroll bar
     const mainContent = document?.querySelector<HTMLElement>(
       `.${baseStyles['main-content']}`
     );
-    let scrollBarWidth =
+    const scrollBarWidth =
       mainContent && mainContent.offsetWidth - mainContent.clientWidth;
 
     if (scrollBarWidth) {
-      scrollBarWidth += 2;
-      const sideButton = document?.querySelector<HTMLElement>(
-        `.${sideButtonStyles['side-button']}`
+      // Set the scroll bar width as a Custom Property for the CSS to use
+      window.document.body.style.setProperty(
+        '--scroll-bar-width',
+        `${scrollBarWidth.toString()}px`
       );
-      const sideButtonHelp = document?.querySelector<HTMLElement>(
-        `.${sideButtonStyles.help}`
-      );
-
-      if (sideButton) {
-        sideButton.style.right = `${scrollBarWidth.toString()}px`;
-      }
-      if (sideButtonHelp) {
-        sideButtonHelp.style.right = `${scrollBarWidth.toString()}px`;
-      }
     }
 
     sleep(3000).then(() => {
       // If there's already Hotjar's feedback, don't do anything
-      if (document.querySelector('._hj_feedback_container')) {
+      if (document.querySelector('#survey_1067707')) {
```

Comment:
As `#survey_1067707` is located in a few places, it could be easier to update in the future by creating a variable and using this instead eg:

```
const surveyId = '#survey_1067707'
```